lexicons/tools/ozone/moderation/defs.json

```diff
@@ -619,6 +630,58 @@
         "height": { "type": "integer" },
         "length": { "type": "integer" }
       }
+    },
+    "accountHosting": {
+      "type": "object",
+      "required": ["status"],
+      "properties": {
+        "status": {
+          "type": "string",
+          "knownValues": ["takendown", "suspended", "deleted", "deactivated"]
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "deletedAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "deactivatedAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "reactivatedAt": {
+          "type": "string",
+          "format": "datetime"
+        }
+      }
+    },
+    "recordHosting": {
+      "type": "object",
+      "required": ["status"],
+      "properties": {
+        "status": {
+          "type": "string",
+          "knownValues": ["takendown", "suspended", "deleted", "deactivated"]
+        },
```

Comment:
From a hosting perspective individual records can't be deactivated, takendown, or suspended— only accounts can.  They can be deleted, though.

Reply:
Ah, I see the values I'd expect reflected in the implementation, so this probably just needs to be updated.